client/my-sites/post-share/index.jsx

```diff
@@ -249,33 +250,33 @@ class PostShare extends Component {
 			businessRawPrice,
 			businessDiscountedRawPrice,
 			userCurrency,
-			sharingActions,
+			scheduledSharingActions,
+			siteId,
+			postId,
 		} = this.props;
 
 		if ( planSlug !== PLAN_BUSINESS ) {
-			return (
-				<Banner
-					className="post-share__footer-banner"
-					callToAction={
-						translate( 'Upgrade for %s', {
-							args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
-							comment: '%s will be replaced by a formatted price, i.e $9.99'
-						} )
-					}
-					description={ translate( 'Live chat support and no advertising.' ) }
-					list={ [
-						translate( 'Live chat support' ),
-						translate( 'No Advertising' )
-					] }
-					plan={ PLAN_BUSINESS }
-					title={ translate( 'Upgrade to a Business Plan!' ) }
-				/>
-			);
+			return ( <Banner
+				className="post-share__footer-banner"
+				callToAction={
+					translate( 'Upgrade for %s', {
+						args: formatCurrency( businessDiscountedRawPrice || businessRawPrice, userCurrency ),
+						comment: '%s will be replaced by a formatted price, i.e $9.99'
+					} )
+				}
+				description={ translate( 'Live chat support and no advertising.' ) }
+				list={ [
+					translate( 'Live chat support' ),
+					translate( 'No Advertising' )
+				] }
+				plan={ PLAN_BUSINESS }
+				title={ translate( 'Upgrade to a Business Plan!' ) }
+			/> );
 		}
-
-		return sharingActions.map(
-			( item, index ) => this.renderFooterSectionItem( item, index )
-		);
+		return ( <div>
+			<QuerySharePostActions siteId={ siteId } postId={ postId } status="scheduled" />
+			{ scheduledSharingActions.map( ( item, index ) => this.renderFooterSectionItem( item, index ) ) }
```

Comment:
> Why are we adding the query component here? Maybe it can be added into the renderFooterSectionItem() method.

It's not necessary as the query component loads all the scheduled and published shares at once so it can be loaded just once.